apps/portlet-documents/src/main/webapp/vue-app/attachment/components/attachments-drive-explorer/AttachmentsDriveExplorerFileItem.vue

```diff
@@ -5,8 +5,16 @@
       rel="tooltip"
       class="fileTitle d-flex flex-column v-messages"
       data-placement="bottom">
-      <div class="fileType center">
-        <i :class="getIconClassFromFileMimeType(file.mimetype)"></i>
+      <div :class="file.isSelected && 'selected'" class="fileType d-flex">
+        <div class="file-type-icon pa-0 col-sm-9 text-right pt-2">
```

Comment:
`col-sm-9` will not be applied on mobile, you may need to use `col-9` instead

```suggestion
        <div class="file-type-icon pa-0 col-9 text-right pt-2">
```